src/Microsoft.AspNet.Antiforgery/DefaultAntiforgery.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.AspNet.Antiforgery
     /// Provides access to the anti-forgery system, which provides protection against
     /// Cross-site Request Forgery (XSRF, also called CSRF) attacks.
     /// </summary>
-    public class Antiforgery
+    public class DefaultAntiforgery : IAntiforgery
```

Comment:
There's no been no decision that I'm aware of this. The Antiforgery code now follows a consistent pattern. If we need to change it later it will be easy.